test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -26,110 +26,94 @@
-    ("aten::slow_conv_transpose2d", datetime.date(2022, 1, 31)),
-    ("aten::slow_conv_transpose2d_backward", datetime.date(2022, 1, 31)),
-    ("aten::slow_conv_transpose3d", datetime.date(2022, 1, 31)),
-    ("aten::slow_conv_transpose3d_backward", datetime.date(2022, 1, 31)),
+ALLOW_LIST_WITH_DEADLINE = [
     ("aten::_svd_helper", datetime.date(2022, 3, 31)),
     ("aten::linalg_svdvals", datetime.date(2022, 3, 31)),
     ("aten::linalg_svdvals_out", datetime.date(2022, 3, 31)),
     ("aten::linalg_svd", datetime.date(2022, 3, 31)),
     ("aten::linalg_svd_out", datetime.date(2022, 3, 31)),
     ("aten::_max_pool1d_cpu_forward", datetime.date(2022, 2, 8)),
-    ("aten::_convolution_nogroup", datetime.date(9999, 1, 1)),
     ("aten::linspace", datetime.date(2022, 3, 1)),  # TODO this will be removed soon
     ("aten::logspace", datetime.date(2022, 3, 1)),  # TODO this will be removed soon
-    ("aten::miopen_convolution_backward", datetime.date(9999, 1, 1)),
-    ("aten::miopen_convolution_backward_bias", datetime.date(9999, 1, 1)),
-    ("aten::miopen_convolution_backward_input", datetime.date(9999, 1, 1)),
-    ("aten::miopen_convolution_backward_weight", datetime.date(9999, 1, 1)),
-    ("aten::miopen_convolution_transpose_backward", datetime.date(9999, 1, 1)),
-    ("aten::miopen_convolution_transpose_backward_input", datetime.date(9999, 1, 1)),
-    ("aten::miopen_convolution_transpose_backward_weight", datetime.date(9999, 1, 1)),
-    ("aten::miopen_depthwise_convolution_backward", datetime.date(9999, 1, 1)),
-    ("aten::miopen_depthwise_convolution_backward_input", datetime.date(9999, 1, 1)),
-    ("aten::miopen_depthwise_convolution_backward_weight", datetime.date(9999, 1, 1)),
-    ("caffe2::", datetime.date(2021, 10, 23)),
-    ("prepacked::unpack_prepacked_sizes_conv2d", datetime.date(9999, 1, 1)),
-    ("prepacked::unpack_prepacked_sizes_linear", datetime.date(9999, 1, 1)),
-    ("q::_FloatToBfloat16Quantized", datetime.date(2021, 12, 21)),
-    ("q::_Bfloat16QuantizedToFloat", datetime.date(2021, 12, 21)),
-    ("aten::_inverse_helper", datetime.date(2021, 12, 31)),
-    ("aten::softplus_backward", datetime.date(2022, 1, 31)),
-    ("aten::softplus_backward.grad_input", datetime.date(2022, 1, 31)),
     ("aten::quantile", datetime.date(2022, 9, 30)),
     ("aten::nanquantile", datetime.date(2022, 9, 30)),
     ("aten::_convolution_double_backward", datetime.date(2022, 3, 31)),
-    ("aten::_scatter_reduce", datetime.date(2022, 1, 31)),
-    ("aten::native_multi_head_self_attention", datetime.date(9999, 1, 1)),
-    ("aten::_native_multi_head_self_attention", datetime.date(9999, 1, 1)),
 ]
 
-ALLOW_LIST_COMPILED = [
-    (
-        re.compile(item[0]),
-        item[1],
-        re.compile(item[2]) if len(item) > 2 else None,
-    ) for item in ALLOW_LIST if item[1] >= datetime.date.today()
+MAX_ALLOWED_PERIOD = datetime.timedelta(days=365)
+
+# WARNING: The operators that are included in this list is extremely unsafe
```

Comment:
This is a little ungrammatical. Maybe something more like: "Operators included in this list indefinitely bypass all BC and FC schema checks. This is almost certainly NOT what you want to do. See note above."?

Reply:
Changed